config/php-cs-fixer.php

```diff
@@ -149,7 +149,7 @@
             // whitespace
             'compact_nullable_typehint' => true,
             'no_extra_blank_lines' => true,
-            'no_spaces_inside_parenthesis' => true,
+            'spaces_inside_parentheses' => true,
```

Comment:
[Allowed values for `spaces_inside_parentheses` are `'none'` or `'single'`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.26.1/doc/rules/whitespace/spaces_inside_parentheses.rst).